lib/cadet/jobs/autograder/lambda_worker.ex

```diff
@@ -44,19 +44,16 @@ defmodule Cadet.Autograder.LambdaWorker do
       "Failed to get autograder result. answer_id: #{answer.id}, error: #{inspect(error)}"
     )
 
-    Que.add(
-      ResultStoreWorker,
-      %{
-        answer_id: answer.id,
-        result: %{
-          grade: 0,
-          status: :failed,
-          errors: [
-            %{"systemError" => "Autograder runtime error. Please contact a system administrator"}
-          ]
-        }
+    Que.add(ResultStoreWorker, %{
```

Comment:
I wonder why are these getting re-formatted by your `mix format`. Can you reply here the output of this command:
```bash
elixir --version
```